core/src/test/java/io/confluent/kafka/schemaregistry/RestApp.java

```diff
@@ -72,6 +72,12 @@ public void stop() throws Exception {
     }
   }
 
+  public void addConfigs(Properties props) {
+    if(props != null) {
```

Comment:
i think that's going to fail checkstyle, needs a space afer the if.

also, wouldn't it just be unexpected to pass null? seems like if anything that should do something like generate an IllegalArgumentException